### ISSUE TITLE
How to set up the Build Server to run as a service

### ISSUE THREAD

**Issue by meliske** [OpenMake-Software/Meister#17]
David is reporting that the build server service (omint) is having the same trouble running as a service that the KB server service did (64-bit Windows platform).  The instructions that you sent using yajsw for the KB server worked perfectly. I’m not finding any instructions for setting up omint as a service.  Can you point me to those instructions, or show us how to use yajsw to set up the build server to run as a service?

**Comment by meliske** [comment]
Found "Build Server as a Windows Service" instructions here: https://www.openmakesoftware.com/help/#!buildserverasawindowsservice.htm

**Comment by sbtaylor15** [comment]
Yes that is the correct instructions for their install.  Let me know if they run into any issues.
